Components/PrizeSVG.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { View, Text } from "react-native";
+import { StyledText } from "./Text";
+
+import Question1 from "../assets/SVGs/myRP/blue-question.svg";
+import Question2 from "../assets/SVGs/myRP/purple-question.svg";
+import Question3 from "../assets/SVGs/myRP/pink-question.svg";
+import Diamond from "../assets/SVGs/myRP/diamond.svg";
+import Sticker from "../assets/SVGs/myRP/sticker.svg";
+import ToteBag from "../assets/SVGs/myRP/tote-bag.svg";
+import Cap from "../assets/SVGs/myRP/cap.svg";
+
+interface PrizeSVGProps {
+  prizeNum: number;
+  attendeePoints: number;
+  token: string;
+}
+
+const FoodWaveSVG: React.FC<PrizeSVGProps> = ({
+  prizeNum,
+  attendeePoints,
+  token,
+}) => {
+  const size = 100;
+
+  const getSVG = () => {
+    if (token) {
+      // Map SVGs based on prizeNum and attendeePoints
+      const svgMap: { [key: number]: JSX.Element } = {
+        1: <Sticker width={size} height={size}/>,
```

Comment:
also i believe the button prize also has a point threshold—this is what I grabbed from Riya's PR:

`"isEligibleMerch.Cap": new_points >= 50,
        "isEligibleMerch.Tote": new_points >= 35,
        "isEligibleMerch.Button": new_points >= 20,`

so let's change the thresholds to these (Riya got these from Oju so I'm assuming these are the correct thresholds)